docs/platforms/flutter/integrations/slow-and-frozen-frames-instrumentation.mdx

```diff
@@ -9,20 +9,21 @@ categories:
 ---
 
 Unresponsive UI and animation hitches annoy users and degrade the user experience.
-Two measurements to track these types of experiences are **slow frames** and **frozen frames**.
-Sentry's slow and frozen frames instrumentation helps track these measurements.
+Three measurements to track these types of experiences are **slow frames**, **frozen frames** and **frames delay**
+Sentry's slow and frozen frames and frames delay instrumentation helps track these measurements.
+Learn more about frames delay [here](https://develop.sentry.dev/sdk/performance/frames-delay/).
```

Comment:
```suggestion
Learn more about frame delay [here](https://develop.sentry.dev/sdk/performance/frames-delay/).
```

Reply:
I know it says "frames delay" in our developer docs, but it should be "frame delay" there too. You can say "frame delay" or "frames delays" because "delay" is singular ;p

Reply:
Yah I was confused about that too, thought it was intentional, thx!